pbf/tests.py

```diff
@@ -215,6 +215,45 @@ def test_round_down_odd_even(self):
     def test_round_down_even_odd(self):
         self.assert_rot(10, 5, 2, round_down=True)
 
+class TestChooseCard(TestCase):
```

Comment:
I had to add tests because I got the logic wrong at first, originally allowing gaining all the majors